imports/plugins/included/default-theme/client/styles/media.less

```diff
@@ -164,6 +164,21 @@
 
 }
 
+.gallery-image.no-fade-on-hover {
```

Comment:
any sense in making this a mixin to use here and above (`.admin-gallery-image.no-fade-on-hover`)?